gobblin-cluster/src/main/java/org/apache/gobblin/cluster/GobblinClusterManager.java

```diff
@@ -544,6 +586,24 @@ public void onLiveInstanceChange(List<LiveInstance> liveInstances, NotificationC
     }
   }
 
+  private class ClusterStandardMetrics extends StandardMetrics {
```

Comment:
Since it's a nested class, I guess `Metrics` is a sufficient name. The reader doesn't need to think about the meaning of "cluster", "standard".